webpack.common.config.ts

```diff
@@ -143,6 +149,15 @@ module.exports = {
   * once angularjs is removed from corresponding pages.
   */
   plugins: [
+    // This plugin performs a direct text replacement, so the value given to it
```

Comment:
Why do we need to do this using webpack?

Reply:
Also, shouldn't this be just in the production config?

Reply:
Went ahead and moved some of the constants to the prod config.